lib/drivers/carelinkDriver.js

```diff
@@ -221,6 +221,12 @@ module.exports = function(simulatorMaker, api){
         progress(20);
 
         function getUnits(colName) {
+          if (colName == null) {
+            var err = new Error('No column labels; check CareLink credentials.');
```

Comment:
So standard [js errors](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error) don't have a code / friendlyMessage. Do you think given that you are throwing a generic error here and then attaching the details to it that it could still be worth having an uploader error type ?